src/Lazy/concurrentPool.ts

```diff
@@ -0,0 +1,179 @@
+import { isAsyncIterable } from "../_internal/utils";
+import type { Reject, Resolve } from "../types/Utils";
+
+type Item<T> = {
+  fail?: unknown;
+  success:
+    | { value: undefined; done: true } // done
+    | { value: T; done: false };
+};
+
+/**
+ * @experimental concurrentPool
+ */
+function concurrentPool<A>(
+  length: number,
```

Comment:
I’m not sure if the name length clearly represents its role. Is this a common variable name in functions dealing with concurrency?

Reply:
I tried to match it with the variable name of [concurrent](https://github.com/marpple/FxTS/blob/main/src/Lazy/concurrent.ts#L66).

When changing it, it would be better to proceed with the variable name of concurrent!
ex) `concurrency`?

Reply:
got it